.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "third-party/gemmi"]
 	path = third-party/gemmi
-	url = git@github.com:project-gemmi/gemmi.git
+	url = https://github.com/project-gemmi/gemmi.git
```

Comment:
Yes, this is probably better

Reply:
It would not let me run git submodule update --init without this fix.